tests/data/hostgaplugin/vm_settings-requested_version.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
The vmsettings API is still in flux and I need to update the data files from time to time. It is very hard to update all those test files, especially when the difference is a single property (as in this case) and that is not pointed out anywhere,

Maybe you can generate those test files on the fly?

Reply:
I dont want to inject this property in a text file and then test that property exists. You can skip updating this file for now, I'll keep modifying it during my PRs and once the feature is enabled I'll make sure to test it out and update this file if needed.

Reply:
the issue is that it'll get parsed so as i make changes in the parsing code i may have to update it

Reply:
@larohra actually you can leave it as is. It just hit me that most of the tests that use the mockwiredata are raising an exception anyways, it's only that we handle the exception by logging it and then we continue (since for now errors in vmsettings are just infos). If I see any issues we can revisit the decision on whether this should be dynamic or not.

Reply:
Sounds good, thanks!